src/pj_init.c

```diff
@@ -415,6 +415,7 @@ pj_init_plus_ctx( projCtx ctx, const char *definition )
 }
 
 
+void *dealloc_params (projCtx ctx, paralist *start, int errlev);
```

Comment:
I'm not sure this makes sense here. If it's used elsewhere, it should be in a header file; if it's only used in this file, then it should be `static` and no prototype is needed.

Reply:
Thanks, @QuLogic - I actually needed `dealloc_params()` (now `pj_dealloc_params()`) in `pj_init_ctx()` and in `pj_default_destructor()`, but had to take a break half way through, as it got very late :-)

`pj_dealloc_params()`is now in `projects.h`